raster/src/main/scala/geotrellis/raster/histogram/ArrayHistogram.scala

```diff
@@ -101,4 +97,11 @@ class ArrayHistogram(val counts: Array[Int], var total: Int)
     }
     return Int.MinValue
   }
+
+  def merge(histogram: Histogram[Int]): Histogram[Int] = {
+    val total = ArrayHistogram(histogram.bucketCount())
+
+    total.update(this); total.update(histogram)
+    total
+  }
 }
```

Comment:
I see you kept ArrayHistogram around...it was on the chopping block, what was the justification for keeping it in?